### PR TITLE
Change Hide Terrain Objects module radius option to a slider

### PR DESCRIPTION
**Pull request by ampersand38**
**When merged this pull request will:**
- Change the radius parameter from edit box to slider 1 to 1000
- Prevents user from setting it too high and crashing their game

### CODE REVIEW

File: addons/modules/functions/fnc_moduleHideTerrainObjects.sqf [hunk 1]
```diff
@@ -55,5 +55,5 @@ deleteVehicle _logic;
 
     {
         [QEGVAR(common,hideObjectGlobal), [_x, _hide]] call CBA_fnc_serverEvent;
-    } forEach nearestTerrainObjects [_position, _objectTypes, parseNumber _range];
+    } forEach nearestTerrainObjects [_position, _objectTypes, parseNumber _radius];
```
Comment (neilzar) [inline addons/modules/functions/fnc_moduleHideTerrainObjects.sqf:58]: `parseNumber` not needed.

**Review by neilzar — CHANGES_REQUESTED** [review]

File: addons/modules/functions/fnc_moduleHideTerrainObjects.sqf (hunk 1 above)
Comment (ampersand38) [reply addons/modules/functions/fnc_moduleHideTerrainObjects.sqf]: aadd2d2

File: addons/modules/functions/fnc_moduleHideTerrainObjects.sqf [hunk 2]
```diff
@@ -26,14 +26,14 @@ deleteVehicle _logic;
         "str_3den_garbagecollection_attribute_mode_displayname",
         [true, 1, 2, ["STR_Disp_Show", "STR_Disp_Hide"]]
     ],
-    ["EDIT", ELSTRING(common,Radius_Units), 10],
+    ["SLIDER", ELSTRING(common,Radius_Units), [1, 100, 10, 0]],
```
Comment (mharis001) [inline addons/modules/functions/fnc_moduleHideTerrainObjects.sqf:29]: Maximum of 100 m seems very low,

Comment (ampersand38) [reply addons/modules/functions/fnc_moduleHideTerrainObjects.sqf:29]: That's kind of hiding a small woods area or city blocks already. I'm certainly open to increasing it to a more useful value.

Comment (CreepPork) [reply addons/modules/functions/fnc_moduleHideTerrainObjects.sqf:29]: I think 500-1000 meters would be nice.

**Review by CreepPork — APPROVED** [review]

File: addons/modules/functions/fnc_moduleHideTerrainObjects.sqf (hunk 2 above)
Comment (mharis001) [reply addons/modules/functions/fnc_moduleHideTerrainObjects.sqf]: 1000 m is probably a good maximum. We can increase it later if needed.

**Review by mharis001 — APPROVED** [review]

**Review by neilzar — APPROVED** [review]

**Pull request merged**